src/components/field-label/field-label.visualroute.js

```diff
@@ -58,5 +59,15 @@ export const component = () => (
         horizontalConstraint="m"
       />
     </Spec>
+    <ThemeProvider theme={themes.darkTheme}>
```

Comment:
```suggestion
      <ThemeProvider theme={themes.darkTheme}>
        <Spec label="with inverted tone">
          <ThemeProvider theme={customProperties}>
            <FieldLabel
              title="Hello"
              description="description"
              hasRequiredIndicator={true}
              tone="inverted"
            />
          </ThemeProvider>
        </Spec>
      </ThemeProvider>
```

Reply:
This way we are actually testing tone -> inverted. Else we are just testing that theming works 😉